fork.yaml

```diff
@@ -5,7 +5,7 @@ footer: |
 base:
   name: go-ethereum
   url: https://github.com/ethereum/go-ethereum
-  hash: b20b4a71598481443d60b261d3e5dcb37f8a0d82  # v1.13.8
+  hash: dd938d103d41f5255a40f7441ad7f0ceae0f7b07  # v1.13.9
```

Comment:
We also need to modify the section at line 245 around the simulated backed

Reply:
done